pkg/local-storage/member/controller/volume_convert_task_worker.go

```diff
@@ -310,3 +347,25 @@ func (m *manager) volumeConvertCleanup(convert *apisv1alpha1.LocalVolumeConvert)
 
 	return m.apiClient.Delete(context.TODO(), convert)
 }
+
+func (m *manager) queryLocalVolume(volumeName string, ctx context.Context, logCtx *log.Entry) (*apisv1alpha1.LocalVolume, error) {
+	vol := &apisv1alpha1.LocalVolume{}
+	if err := m.apiClient.Get(ctx, types.NamespacedName{Name: volumeName}, vol); err != nil {
+		logCtx.WithError(err).Error("Failed to query volume", volumeName)
+		return nil, err
+	}
+	return vol, nil
+}
+
+func (m *manager) queryLocalVolumeGroup(volumeGroupName string, ctx context.Context, logCtx *log.Entry) (*apisv1alpha1.LocalVolumeGroup, error) {
+	lvg := &apisv1alpha1.LocalVolumeGroup{}
+	if err := m.apiClient.Get(ctx, types.NamespacedName{Name: volumeGroupName}, lvg); err != nil {
+		if !errors.IsNotFound(err) {
```

Comment:
why handle `NotFoundError` here. May return error directly and leave it handled at where it called.